crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -34,6 +34,7 @@ def upgrade
       case @version
       when "1.9"
         knife_files_1_9
+        reset_target_platform_for_nodes
```

Comment:
Doh, I missed the creation of the new method despite reading this about 5 times.  OTOH this is a good example of why it's better to do a single refactoring per commit :-)